File: Practice/Basic Programming/Input-Output/py/Factorial.py
# You have been given a positive integer N. You need to find and print the
# Factorial of this number. The factorial of a positive integer N refers to the
# product of all number in the range from 1 to N.
factorial = 1
number = int(input())
# calculating factorial of the number
for i in range(1, number + 1):
	factorial *= i
print(factorial)

File: Practice/Basic Programming/Input-Output/cpp/Factorial.cpp
/* You have been given a positive integer N. You need to find and print the
   Factorial of this number. The factorial of a positive integer N refers to the
   product of all number in the range from 1 to N.
*/
#include<iostream>
using namespace std;
int main() {
	int number, factorial = 1;
	cin >> number;
	// calculating factorial of the number
	for (int i = 1; i <= number; i++) {
		factorial *= i;
	}
	cout << factorial;
	return 0;
}
